src/zjs_ocf_client.c

```diff
@@ -863,14 +863,20 @@ static void ocf_get_platform_info_handler(oc_client_response_t *data)
                 break;
             case BYTE_STRING:
             case STRING:
-                /*
-                 * TODO: try and find the names of all the platform info properties
-                 *       can set a map in C array to reference these strange names
-                 */
                 if (strcmp(oc_string(rep->name), "mnmn") == 0) {
                     zjs_obj_add_string(platform_info, oc_string(rep->value.string), "manufacturerName");
                 } else if (strcmp(oc_string(rep->name), "pi") == 0) {
                     zjs_obj_add_string(platform_info, oc_string(rep->value.string), "id");
+                } else if (strcmp(oc_string(rep->name), "mnmo") == 0) {
```

Comment:
Changing them has no effect but perhaps they should be. Do we have an easy API for that?